extra-network/qv2ray/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=Qv2ray
+PKGSEC=net
+PKGDEP="v2ray qt-5 openssl protobuf gprc"
```

Comment:
Please double check if dependency is correct and complete - as we have discussed before.

Reply:
Working on it

Reply:
Double checked.